doc/source/examples/extended_examples/sfem/sfem.py

```diff
@@ -0,0 +1,758 @@
+
+    def eigen_value(w_n):
+        return (2 * correl_length_param) / (1 + (correl_length_param * w_n) ** 2)
+
+    def sine_constant(w_n):
+        return 1 / (A - (math.sin(2 * w_n * A) / (2 * w_n))) ** 0.5
+
+    # Build the array of eigenvalues and constant terms for the accuracy required
+    for n in range(1, 100):
+        # start solving here
+        acceptable_solution_error = 1e-10
+        solution_range = [(n - 0.5) * math.pi / A, n * math.pi / A]
+        solution = find_solution(
+            func, deriv_func, acceptable_solution_error, solution_range
+        )
+
+        frequency_array.append(solution)
+        sine_eigen_values_array.append(eigen_value(solution))
+        sine_constants_array.append(sine_constant(solution))
+        if eigen_value(solution) < min_eigen_value:
+            break
+
+    return (
+        np.array(frequency_array),
+        np.array(sine_eigen_values_array),
+        np.array(sine_constants_array),
+    )
+
+
+def stochastic_field_realization(
+    cosine_frequency_array: np.ndarray,
+    cosine_eigen_values: np.ndarray,
+    cosine_constants: np.ndarray,
+    cosine_random_variables_set: np.ndarray,
+    sine_frequency_array: np.ndarray,
+    sine_eigen_values: np.ndarray,
+    sine_constants: np.ndarray,
+    sine_random_variables_set: np.ndarray,
+    domain: Tuple[float, float],
+    evaluation_point: float,
+) -> float:
+    """The realization of the gaussian field f(x)
+
+    Parameters
+    ----------
+    cosine_frequency_array : np.ndarray
+        Array of length P, containing frequencies associated with retained cosine terms
+    cosine_eigen_values : np.ndarray
+        Array of length P, containing eigenvalues associated with retained cosine terms
+    cosine_constants : np.ndarray
+        Array of length P, containing constants associated with retained cosine terms
+    cosine_random_variables_set : np.ndarray
+        Array of length P, containing random variable drawn from N(0,1) for the cosine terms
+    sine_frequency_array : np.ndarray
+        Array of length Q, containing frequencies associated with retained sine terms
+    sine_eigen_values : np.ndarray
+        Array of length Q, containing eigenvalues associated with retained sine terms
+    sine_constants : np.ndarray
+        Array of length Q, containing constants associated with retained sine terms
+    sine_random_variables_set : np.ndarray
+        Array of length P, containing random variable drawn from N(0,1) for the sine terms
+    domain : Tuple[float, float]
+        Domain over which the KL representation of the stochastic process should be found
+    evaluation_point : float
+        Point within the domain at which the value of a realization is required
+
+    Returns
+    -------
+    float
+        The value of the realization at a given point within the domain
+    """
+    # Shift parameter -> Because we had solved for terms in a symmetric domain [-A, A]
```

Comment:
```suggestion
    # Shift parameter -> Because terms are solved in a symmetric domain [-A, A]
```
Avoid "we" and use "you" if not rewording in some other way.